pkgs/development/tools/misc/pwndbg/default.nix

```
@@ -7,17 +7,20 @@
 }:
 
 let
-  pythonPath = with python3.pkgs; makePythonPath [
+  venv = python3.withPackages (p: with p; [
```

```suggestion
  pyEnv = python3.withPackages (ps: with ps; [
```
that's more standard naming